CMakeLists.txt

```diff
@@ -12,6 +12,7 @@ option(ENABLE_TESTING "Build with unit tests" OFF)
 option(EXTENSIVE_WARNINGS "Build with all warnings" ON)
 option(ENABLE_MORE_COMPILER_OPTIMIZATION_FLAGS "Enable more optimization flags" ON)
 option(USE_SYSTEM_LIBS "Use the system libraries if available" OFF)
+option(RETARDED_APPLE_CLANG "Apple Clang <= 13 used" OFF)
```

Comment:
Fine with changing it. I do like to note that the usage of "retarded" wasn't used in a degenerative context but actual because it actually covered the meaning of the variable quite well.

![image](https://github.com/Ultimaker/CuraEngine/assets/8535734/d206374e-0de4-4fff-ae4f-ae6d124bea79)